### ISSUE TITLE
Add Surface viewer

### ISSUE THREAD

**Issue by AntoineDao** [ladybug-tools/spider-react#4]
Add a small bit of UI that facilitates viewing details/data about highlights or clicked on surfaces.

**Comment by theo-armour** [comment]
@AntoineDao 

Code looks amazing / everything re-factored.

Will start having a look - and commenting - in a few hours.

**Comment by AntoineDao** [comment]
Yeah it's quite different to the previous setup. The [core](https://github.com/ladybug-tools/spider-core) package contains most of the ThreeJS / gbxml parsing code, whereas this is a react project. I'm happy to have a chat on slack or whatnot about react/webpack and co if it's not something you've come across before so you can have a proper play with it :smile:

**Comment by theo-armour** [comment]
@AntoineDao 

All good. And it might also be fun to see/meet each other. Are you on Google Hangouts or something?

**Comment by AntoineDao** [comment]
Aye indeed. I can do hangout or slack or what not. Shall we organise ourselves on slack or the forum?

**Comment by theo-armour** [comment]
@AntoineDao 

Good to close?

**Comment by AntoineDao** [comment]
Nope. Haven't started on this at all.

**Comment by theo-armour** [comment]
@AntoineDao 

Oh, I'm sorry I cluttered this issue.

My bad, I thought https://www.ladybug.tools/spider-react/ was what you were referring to and therefore the issue had been dealt with. Now reading more carefully, I see you want to add more interaction.
